src/Routing/ResolvedRoute.php

```diff
@@ -4,11 +4,11 @@
 
 use stdClass;
 
-class ResolvedRoute
+readonly class ResolvedRoute
 {
     public function __construct(
-        public readonly Route $route,
-        public readonly stdClass $args
+        public Route    $route,
+        public stdClass $args
```

Comment:
Hmm why did you drop the read-only?

Reply:
I've made the class itself to be `readonly`, class properties will inherit this initializer.

Reply:
Ohh, I didn't even see that. My fault